src/query/service/src/interpreters/interpreter_table_vacuum.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 Datafuse Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+
+use common_exception::ErrorCode;
+use common_exception::Result;
+use common_sql::plans::VacuumTablePlan;
+
+use crate::interpreters::Interpreter;
+use crate::pipelines::PipelineBuildResult;
+use crate::sessions::QueryContext;
+
+#[allow(dead_code)]
+pub struct VacuumTableInterpreter {
+    ctx: Arc<QueryContext>,
+    plan: VacuumTablePlan,
+}
+
+impl VacuumTableInterpreter {
+    pub fn try_create(ctx: Arc<QueryContext>, plan: VacuumTablePlan) -> Result<Self> {
+        Ok(VacuumTableInterpreter { ctx, plan })
+    }
+}
+
+#[async_trait::async_trait]
+impl Interpreter for VacuumTableInterpreter {
+    fn name(&self) -> &str {
+        "VacuumTableInterpreter"
+    }
+
+    #[async_backtrace::framed]
+    async fn execute2(&self) -> Result<PipelineBuildResult> {
+        return Err(ErrorCode::PermissionDenied(
```

Comment:
I think we should define a `Commercial License` code and the related warning message. Any suggestions on these? @ZhiHanZ

Reply:
fix `PermissionDenied` error code to `LicenceDenied`